### ISSUE TITLE
New Mod Request: FindIt

### ISSUE THREAD

**Issue by mitchej123** [GTNewHorizons/NewHorizons#3279]
I couldn't find my EV mixer... which I swore I had built and placed; so I made a mod to help that situation in the future.  It's inspired by the Extra Utilities 'search nearby inventories', but instead of searching inventories it searches tile entities in the world with support for GregTech tile entities (support was _not_ added for GT ores).

https://github.com/mitchej123/FindIt

**Comment by boubou19** [comment]
is it only client side?

**Comment by mitchej123** [comment]
Server + Client; the server needs to run the location of the items.

**Comment by boubou19** [comment]
i'd like to see it in the pack because it happened to me so much times, and still happens to me because i have a too large base.

**Comment by DualBallers** [comment]
Currently I think the default search inventory key is 't'. 
Will this mod use the same key bind, and will both a placed entity and the same item in an inventory show up simultaneously if I search for that?

**Comment by noobyaran9** [comment]
>Currently I think the default search inventory key is 't'.

The search function is provided by NEI which u can remap the key inside NEI in game config.

**Comment by DualBallers** [comment]
Yes, but the question is rather - if the NEI key bind is changed to something else, will the 'search entity' then also use this? And can the item and inventory search work concurrently.

**Comment by Dream-Master** [comment]
Ok will add the mod to my dev inviroment.

**Comment by Bryfer** [comment]
Can you Blacklist items with this search function?
-Like emerald ores, and other non-gt ores
-Firestone and similar blocks?
What search radius does it have and does it have?
-1 Chunk? Or entire base? Does it actually help finding the placed machines?

**Comment by WarlordWossman** [comment]
If it can find chests it can probably also find spawners (be it the spawner or the chests nearby), but I guess that's not the biggest issue since finding a rogue-like dungeon is fairly easy and those have tons of spawners.

**Comment by mitchej123** [comment]
The search inventories functionality is added to nei by extra utilities (hot key T), and I did not duplicate it.  The hot key for this is Y and it only searches for places blocks.  If desired I could add it and have one key for both but currently opted not to duplicate functionality.

The range is configurable and defaults to a 32x32x32 cube (config radius of 16 - badly named but it works).  It does help finding machines which is why I made it :)

Adding a blacklist shouldn't be too hard.  I will try later and see if it works on spawners or emerald or and if so add blacklist functionality.  It works on diamond chests but it didn.t on vanilla chests (at least not double chests).  My main concern was gt ores so I made sure that wasn.t included.  


Will post a compiled version when I get home for people to try.
